data/dapis/polygon-testnet.json

```diff
@@ -1,32 +1,25 @@
-  "LTC/USD": "0x79e3268fa44de913c648260d7d483875ab0095ea18d63be088ef6894bd8bcacd",
-  "MATIC/USD": "0x5f9d68d2108556123524a6d64531bb136676e2ca6effda812cca70e5ef655528",
-  "MIOTA/USD": "0xf94ee8a799f14c1c4900335d4eb73395e224a6cf8ec3fde7734227235a5bd1c8",
-  "NEO/USD": "0xb43559f0ce38988d695fa1a07becebc6add4295ff086fd326d831be92067c240",
-  "OOKI/ETH": "0xc22d30e582730222b2f3227f357452c2275520339e76d831784ed484493b3b1b",
-  "QUICK/USD": "0x31c537609bfa0a2aa1431a607982414963b867f18eabdbaa953174e0c0ce243f",
-  "RBTC/BTC": "0x89277198683adedbe9d0cef363aa6eb53b2fb2dcd7caf06eddfc260ddefe8fec",
-  "RBTC/USD": "0xf04547df734060189a890100e802b6c24c2c2ef184875529e7beed034de39122",
-  "SOL/USD": "0x9d6263fcc0742b5da22e1a8c3677843e4b9cf8b75d0939844c29bdc130c67d9d",
-  "UNI/USD": "0xed3c55fd0d666ca78192ae861f32bc59d0abcb32de417f84a4de8d1a08928120",
-  "USDC/USD": "0x965a0183d076f8088229bca7e90cbfcb7155339b4066de627bb76d8ddc9b03bc",
-  "VET/USD": "0x43e4b505b0bc564a19de751a136cdeb34b90261c2d14700d6ee6c8fdfeea56f4",
-  "XLM/USD": "0x0b819cb4816b1b0ca50cab57f65532ff8abf12920a19f5d16fefe1e48f472741",
-  "XVS/USD": "0xa32c9aba0d4fe040776361503784e1c485615ecd93421712a051fe0f512555c1"
+  "AAVE/USD": "0x55a375e3478936a656c022a9554c45509331bd4d4b647ec3c78c07c5c17237fb",
+  "ADA/USD": "0x712e1ba1be510e1b7d596a12492de4df51e636e835727f782ef00ffa783a7be9",
+  "AVAX/USD": "0xc197aeec5e904b487c37360d0044097a925037f038c6d2b7d92ad20bc689539e",
+  "AXS/USD": "0x8b704cd69414bd302b59a6775595d210555ec490ff2e50ee319f37f7fe04b081",
+  "BNB/USD": "0x0343be29ba1b446ac36f813b2827e207666029f47c168a42ef163bd2bb815451",
+  "BTC/USD": "0x0dc124b07cc935112d87b49c806c5c880659dd7a2ef75b4ea04460cf224ea2c0",
+  "BUSD/USD": "0xe6bc56045455ca1aebc85ddd7a251ecf55b11c0797c53f2833d7caa6b1a91fa0",
+  "CRV/USD": "0x2d7d6ad333ae8f39f8eabe31fd74523178ad8858da126cc9178a1e9138e950a3",
+  "DOT/USD": "0x118e19f765c78f880a575b7e193f5d3e9c73d71b944caf83124e670480460712",
+  "EOS/USD": "0xcdd894ea6d7deb8828df537219e76dc037cfb98f35de34d0759248b6c0734cb4",
+  "FIL/USD": "0xa8d8864ab62909f3590ddf79ee79d5d7386f61b664291388d9bfe715783ea084",
+  "LINK/USD": "0x7c72d2ac334b8af0c215b201cd94d8cb6154688d821d967d42893f23d434f378",
+  "LTC/USD": "0x4c01ad2e75dfd0091c7c9d15507be3c63658172e578209b18bd3e15d820a31a4",
+  "MATIC/USD": "0xe295b7b5ad29e1fdbd256c35af0e003e30c1f500720a4e3bd2c689022a8ba4b3",
+  "MIOTA/USD": "0xdb997781aa3f48c577fbee55c2a746f86fa63068ca0b7a4c444fa6114a5f2465",
+  "NEO/USD": "0xffa121bc7111056fbe457c2c4200b9c6fd9f50edeb503c28f460ce0345080bc6",
+  "SOL/USD": "0xaabf6d9cbaad9f4c60527162747a5108af824964f4fd477734de7aae260c2bde",
+  "UNI/USD": "0x9044a9001d5667f3a3208c571a06c8281c31ec12e18a846b1d5e69cbff531b7a",
+  "USDC/USD": "0xb1b031f4b9616ad61070e27e192f1d57da9dc26dc67712049c80bd0075f913d6",
+  "VET/USD": "0xf2f05dc7deefcc0d3fed542829a9de8f73023a710348e70ec8bf3b88c98105e4",
+  "XLM/USD": "0x89fdbaa2f825578b6be3f6a052c291828508c770a885ffe6ae241e79188339b2",
+  "XVS/USD": ""
```

Comment:
I think, this shouldn't be here?

Reply:
Removed in https://github.com/api3dao/operations/pull/118/commits/e6b1d8faef7d0cc30176427604ecbc76501b2490